components/history/core/browser/sync/brave_typed_url_sync_bridge.h

```diff
@@ -0,0 +1,31 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_HISTORY_CORE_BROWSER_SYNC_BRAVE_TYPED_URL_SYNC_BRIDGE_H_
+#define BRAVE_COMPONENTS_HISTORY_CORE_BROWSER_SYNC_BRAVE_TYPED_URL_SYNC_BRIDGE_H_
+
+#include <memory>
+
+#include "components/history/core/browser/sync/typed_url_sync_bridge.h"
+
+namespace history {
+
+class BraveTypedURLSyncBridge : public TypedURLSyncBridge {
+ public:
+  explicit BraveTypedURLSyncBridge(
```

Comment:
`explicit` is not necessary.

Reply:
Fixed at https://github.com/brave/brave-core/pull/16705/commits/875844cd3a0e02a8c06a780660fd45e1dd4bae3a